flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/router/RoutedRequest.java

```diff
@@ -94,4 +94,14 @@ public ReferenceCounted retain(int arg0) {
 		}
 		return this;
 	}
+
+	@Override
+	public ReferenceCounted touch() {
+		return this;
+	}
+
+	@Override
+	public ReferenceCounted touch(Object o) {
+		return this;
```

Comment:
We may want to `touch` the contained request as well like in [this](https://github.com/netty/netty/blob/4.1/codec-redis/src/main/java/io/netty/handler/codec/redis/ArrayRedisMessage.java) class, depending on `requestAsReferenceCounted`.